archive/s/swift/fizzbuzz.swift

```diff
@@ -0,0 +1,22 @@
+func fizzBuzz(start: Int = 1, end: Int = 100) -> Void {
```

Comment:
So, I see this function which looks great. Can you make the script execute this function? In other words, the solution should be executable.

Reply:
I tested this over in the Swift playground, and it looks like you can just call fizz buzz at the end.

Reply:
That's right. The function uses default parameters, so you can run it just by calling it without any arguments, and it will know to go through all the numbers between 1 and 100. You can also pass in values for start and end when you call it.

Reply:
Can you update the solution to make a call to fizz buzz at the bottom?